src/app/screens/Detail/styles.ts

```diff
@@ -0,0 +1,68 @@
+import { StyleSheet } from 'react-native';
+import { COLORS } from '@constants/colors';
```

Comment:
i think that you can export each color by itself so you dont have yo import all the colors. In this case maybe this change is not that important since you are using a lot of colors but other cases are not like this one.